components/fieldgroups/intent/demandbase-account-intent-details.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/demandbase-account-intent-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Demandbase Account Intent Details",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/demandbase-account-intent"
+  ],
+  "description": "Captures and maintains additional demandbase intent attributes.",
+  "definitions": {
+    "demandbase-intent-details": {
+      "properties": {
+        "xdm:intentID": {
+          "title": "Intent ID",
+          "description": "Adobe generated unique identifier for the record.",
+          "type": "string"
+        },
+        "xdm:accountId": {
```

Comment:
Pl. change `xdm:accountId` to `xdm:accountID` to keep it consistent with the naming conventions.